src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -242,9 +251,13 @@ private J.Block rewriteTemplate(String verifyTemplate, List<Object> templatePara
                 );
     }
 
-    private static @Nullable String getWhenTemplate(List<Expression> results) {
+    private @Nullable String getWhenTemplate(List<Expression> results) {
```

Comment:
Can we remove `@Nullable`? I mentioned this in a previous PR but it's essentially tech debt unless you go straight to jakarta.

I noticed that we are using openrewrite.lang.internal package, but internal typically means it's not intended for users but rather the framework itself and is at risk to breaking changes.

Reply:
I don't want to use Nullable, I just can't think of what to replace Expression times with. Any suggestions?

Reply:
I need to add dependency for jakarta as not there, happy with this? Tried to reduce dependency on @Nullable as much as possible, but there is one case where it is being used as an argument. I could convert it to an Optional, let me know if there's a better option.